$OpenBSD: patch-Rakefile,v 1.1 2011/11/17 16:08:00 jeremy Exp $

The gem doesn't ship with it's gemspec, which breaks the Rakefile
when used from the gem.  Work around it so the regress tests can run.

--- Rakefile.orig	Sun Nov 13 22:29:39 2011
+++ Rakefile	Sun Nov 13 22:29:54 2011
@@ -10,7 +10,6 @@ require 'date'
 # rake cleanr default
 # gem push pkg/ruby-prof-0.10.1.gem
 
-default_spec = eval File.read(File.expand_path('../ruby-prof.gemspec', __FILE__))
 
 desc 'deprecated--build native .gem files -- use like "native_gems clobber cross native gem"--for non native gem creation use "native_gems clobber" then "clean gem"'
 task :native_gems do
@@ -27,7 +26,7 @@ end
 Rake::GemPackageTask.new(default_spec) do |pkg|
   pkg.need_tar = true
   #pkg.need_zip = true
-end
+end if false
 
 
 # ---------  RDoc Documentation ------
